training_config/generation/t5_11b_cnn_dm.jsonnet

```diff
@@ -0,0 +1,125 @@
+// =================== Configurable Settings ======================
+
+// In 'debug' mode, we only train t5-small over a few instances on 2 GPUs.
+// You probably want to start out on debug mode (the default) before attempting a full
+// training run.
+//
+// When debug=false, we train t5-11b on 8 GPUs (less than 8 GPUs probably
+// won't give you enough memory).
+//
+// You could also adapt this config to train on smaller t5 varients, in which case
+// you may not need to use the "fairscale_fsdp" DdpWrapper or the "fairscale" CheckpointWrapper.
+local debug = true;
+
+// This is probably necessary for t5-11b unless you have more than 8 GPUs.
+local activation_checkpointing = true;
+
+// Set to `false` if you want to skip validation.
+local validate = true;
+
+// AMP is currently unusably slow with t5-11b, which be due to a bug bug within
+// FairScale, but I'm not sure yet.
+local use_amp = false;
+
+// These are reasonable defaults.
+local source_length = 512;
+local target_length = 54;
+
+// Only set to `true` if you're running this on Beaker batch.
+local on_beaker = false;
+
+// ================================================================
+
+// ------ !! You probably don't need to edit below here !! --------
+
+local model_name = if debug then "t5-small" else "t5-11b";
+local batch_size_per_gpu = if debug then 4 else 1;
+
+local data_base_url = "https://storage.googleapis.com/allennlp-public-data/cnndm-combined-data-2020.07.13.tar.gz";
+local train_data = data_base_url + "!cnndm-combined-data-2020.07.13/url_lists/all_train.txt";
+local dev_data = data_base_url + "!cnndm-combined-data-2020.07.13/url_lists/all_val.txt";
+
+local dataset_reader = {
+    "type": "cnn_dm",
+    "source_tokenizer": {
+        "type": "pretrained_transformer",
+        "model_name": model_name,
+    },
+    "source_token_indexers": {
+        "tokens": {
+            "type": "pretrained_transformer",
+            "model_name": model_name,
+            "namespace": "tokens",
+        }
+    },
+    "source_max_tokens": source_length,
+    "target_max_tokens": target_length,
+    "source_prefix": "summarize: ",
+};
+
+local data_loader = {
+    "batch_size": batch_size_per_gpu,
+    "shuffle": true,
+};
+
+local wandb_callback = {
+    "type": "wandb",
+    "project": "allennlp-t5",
+    "entity": "allenai-team1",
+    "watch_model": false,
+    "summary_interval": 1,
+    "should_log_parameter_statistics": false,
+    "should_log_learning_rate": false,
+};
```

Comment:
I don't know if we want this in a default config. Virtually everyone using this will have to change it.

Though I would be entirely in favor of having a commented-out wandb config in every jsonnet file.